src/main/java/net/glowstone/entity/EntityManager.java

```diff
@@ -93,6 +96,7 @@ void register(GlowEntity entity) {
      * @param entity The entity.
      */
     void unregister(GlowEntity entity) {
+        EventFactory.getInstance().callEvent(new EntityRemoveFromWorldEvent(entity));
```

Comment:
Shouldn't this be checked for cancellation?

Reply:
Neither the event nor any of its superclasses implement `Cancellable`